### PR TITLE
refactor listenDirEvent

### PR DESCRIPTION
**Pull request by Mulavar**
1. remove CLEAR label;
2. remove register/unregister logic for watch children;
3. reorder the zk dynamic configuration code
4. adapt the gost

**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: remoting/zookeeper/listener.go [hunk 1]
```diff
@@ -366,7 +346,7 @@ func (l *ZkEventListener) startScheduleWatchTask(
 				ticker = time.NewTicker(tickerTTL)
 			}
 		case zkEvent := <-childEventCh:
-			logger.Warnf("get a zookeeper childEventCh{type:%s, server:%s, path:%s, state:%d-%s, err:%v}",
+			logger.Debugf("get a zookeeper childEventCh{type:%s, server:%s, path:%s, state:%d-%s, err:%v}",
```
Comment (XiaoWeiKIN) [inline remoting/zookeeper/listener.go:349]: Get 大写

File: registry/zookeeper/registry.go [hunk 2]
```diff
@@ -150,7 +150,11 @@ func (r *zkRegistry) InitListeners() {
 
 // CreatePath creates the path in the registry center of zookeeper
 func (r *zkRegistry) CreatePath(path string) error {
-	return r.ZkClient().Create(path)
+	err := r.ZkClient().Create(path)
+	if err != nil && err != zk.ErrNodeExists {
```
Comment (justxuewei) [inline registry/zookeeper/registry.go:154]: IMO, using errors.Is() is better than `!=`.

File: config_center/zookeeper/impl.go [hunk 3]
```diff
@@ -80,15 +81,20 @@ func newZookeeperDynamicConfiguration(url *common.URL) (*zookeeperDynamicConfigu
 		logger.Errorf("zookeeper client start error ,error message is %v", err)
 		return nil, err
 	}
+	err = c.client.Create(c.rootPath)
+	if err != nil && err != zk.ErrNodeExists {
```
Comment (justxuewei) [inline config_center/zookeeper/impl.go:85]: errors.Is

File: registry/zookeeper/registry.go (hunk 2 above)
Comment (LaurenceLiZhixin) [reply registry/zookeeper/registry.go:154]: I think != is just ok

**Review by LaurenceLiZhixin — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Review by justxuewei — APPROVED** [review]

**Pull request merged**